microsite/sidebars.js

```diff
@@ -496,30 +505,54 @@ module.exports = {
       'tutorials/enable-public-entry',
       'tutorials/setup-opentelemetry',
     ],
-    'Architecture Decision Records (ADRs)': [
-      'architecture-decisions/adrs-overview',
-      'architecture-decisions/adrs-adr001',
-      'architecture-decisions/adrs-adr002',
-      'architecture-decisions/adrs-adr003',
-      'architecture-decisions/adrs-adr004',
-      'architecture-decisions/adrs-adr005',
-      'architecture-decisions/adrs-adr006',
-      'architecture-decisions/adrs-adr007',
-      'architecture-decisions/adrs-adr008',
-      'architecture-decisions/adrs-adr009',
-      'architecture-decisions/adrs-adr010',
-      'architecture-decisions/adrs-adr011',
-      'architecture-decisions/adrs-adr012',
-      'architecture-decisions/adrs-adr013',
-    ],
+
     FAQ: ['faq/index', 'faq/product', 'faq/technical'],
     Accessibility: ['accessibility/index'],
     Contribute: [
       'contribute/index',
       'contribute/getting-involved',
       'contribute/project-structure',
     ],
-    References: ['references/glossary'],
+    References: [
+      'references/glossary',
+      {
+        type: 'category',
+        label: 'API Reference',
+        items: [
+          {
+            type: 'category',
+            label: 'Guides',
+            items: ['api/utility-apis'],
+          },
+          {
+            type: 'category',
+            label: 'API Reference',
+            items: ['reference/index'],
+          },
+          'api/deprecations',
+        ],
```

Comment:
I think we can flatten these out into the parent, i.e. all of these directly under "References"

Reply:
Updated as well, I think some of these have a better home elsewhere - utility APIs seems related to the (old?) frontend system and deprecations has 2 entries from years ago. Going to keep them here, but worth coming back to when we want to do a more in depth rewrite.